### DISCUSSION TITLE
Why delete internal docs in master

### DISCUSSION THREAD

**Discussion by HackToday** [apache/doris#9310] [A - General / Q&A] [answered]
I checked latest master branch code, the 
https://github.com/apache/incubator-doris/tree/1.0.0-rc03/docs/zh-CN/internal

not exist anymore. Does anyone know the reason for such deletion?  Thanks

**Comment by hf200012** [comment]
The document structure has been refactored

**Reply by HackToday** [reply]
@hf200012  Where can find the internal doc in master branch ? I did not find that in the docs.  Thanks

**Comment by HackToday** [comment, marked as answer]
It has been added in https://github.com/apache/incubator-doris/tree/master/docs/zh-CN/design in lastest commit.
